kubernetes-on-ubuntu-vmss/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"vmName": {
+			"value": "GEN-UNIQUE"
+		},
+		"vmssName": {
+			"value": "GEN-UNIQUE"
+		},
+		"adminUsername": {
+			"value": "GEN-UNIQUE"
+		},
+		"adminKey": {
+			"value": "GEN-SSH-PUB-KEY"
+		},
+		"minNodeCount": {
```

Comment:
remove the non-GEN values and add them as defaultValues in the template

Reply:
moved all non-GEN values to template